language/bytecode_verifier/src/type_memory_safety.rs

```diff
@@ -70,7 +70,16 @@ impl<'a> TypeAndMemorySafetyAnalysis<'a> {
             errors: vec![],
         };
 
-        verifier.analyze_function(initial_state, &function_definition_view, cfg);
+        let inv_map = verifier.analyze_function(initial_state, &function_definition_view, cfg);
+        // Report all the join failures
+        for (block_id, inv) in inv_map.iter() {
+            match inv.post() {
+                BlockPostcondition::Error => verifier
```

Comment:
Could you describe what exactly is going on here? How does this represent a failure at a join?

Reply:
Good catch; this actually represents all errors, not just join failures. Let me fix that.